test/optimization/test_converters.py

```diff
@@ -40,6 +65,46 @@ def test_empty_problem(self):
         qubitop, shift = conv.encode(op)
         self.assertEqual(shift, 0.0)
 
+    def test_valid_variable_type(self):
+        # validate the types of the variables for OptimizationProblemToOperator
```

Comment:
If you convert this to a docstring it counts as function docstring and will not raise a pylint warning.
```suggestion
    def test_valid_variable_type(self):
        """Validate the types of the variables for OptimizationProblemToOperator."""
```